corehq/apps/userreports/views.py

```diff
@@ -220,7 +221,11 @@ def dispatch(self, request, *args, **kwargs):
                 'worker': ConfigureWorkerReport,
                 'table': ConfigureTableReport
             }[report.report_meta.builder_report_type]
-            return view_class.as_view(existing_report=report)(request, *args, **kwargs)
+            try:
+                return view_class.as_view(existing_report=report)(request, *args, **kwargs)
+            except Exception, e:
```

Comment:
`except Exception:` would fix this particular PEP8 comment. `except Exception as e:` is preferred over `except Exception, e` in recent (i.e. last 5 years) versions of python